bin/assume-aws-role.js

```diff
@@ -70,6 +72,25 @@ if (command == "list") {
   	process.exit(0);
 }
 
+if (command == "delete") {
+	var alias = process.argv[3];
+	if (!alias) {
+	    console.error("No alias specified");
+	    console.error("Usage: assume-aws-role delete <alias>");
+	    process.exit(1);
+	}
+	var config = readConfig(filename);
+	var role = config[alias];
+	if (!role) {
+	    console.error("The specified alias does not exist");
+	    process.exit(1);
+	}
+	delete config[alias];
+	fs.outputJsonSync(filename, config);
+	console.log("The alias %s was deleted", alias);
```

Comment:
Could you remove this line? Generally I try to only print stuff when something fails